src/Private/Context.ps1

```diff
@@ -938,6 +938,13 @@ function Set-PodeWebConfiguration {
         }
     }
 
+    if ($Configuration.OpenApi -and $Configuration.OpenApi.ContainsKey('UsePodeYamlInternal')) {
+        if (!$Context.Server.Web.ContainsKey('OpenApi')) {
+            $Context.Server.Web.OpenApi = @{}
+        }
+        $Context.Server.Web.OpenApi.UsePodeYamlInternal = $Configuration.OpenApi.UsePodeYamlInternal
+    }
```

Comment:
Any reason to not move this up to the main web config hashtable definition above, as the following, just below the `Compression = @{`? It should equate to the same, and default to `$false` as well

```powershell
OpenApi = @{
    UsePodeYamlInternal = [bool]$Configuration.OpenApi.UsePodeYamlInternal
}
```

Reply:
because the  $Context.Server.Web.OpenApi, in theory, can be an existing key

Reply:
`$Context.Server.Web` gets created for the first time in the block just above this one, at line 910, so the `OpenApi` key wouldn't exist at this point

Reply:
Let me check probably you are right